src/main/resources/data/alaskanativecraft/recipes/dry_ptarmigan.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ingredient": {
+    "item": "alaskanativecraft:cooked_ptarmigan"
```

Comment:
drying generally uses raw ingredients, not cooked ones (this comment applies to all the other drying recipes I just don't want to spam with duplicate comments)

Reply:
I can do that, yes